dangerzone/document.py

```diff
@@ -103,6 +103,31 @@ def default_output_filename(self) -> str:
 
     def announce_id(self) -> None:
         log.info(f"Assigning ID '{self.id}' to doc '{self.input_filename}'")
+    def set_output_dir(self, path: str) -> None:
```

Comment:
Should we check at some point if the output filenames from different dirs end up being the same name. This can happen if we have the same filename in different dirs.

Reply:
I don't think we need that at this point because the user should only be able to select from one directory. At least I am assuming that.

The one case that might affect us is if we have `file.pdf` and `file.gif`. Both will converge to the same one. This actually happens on the sample test files. So we should catch this.

Reply:
> I don't think we need that at this point because the user should only be able to select from one directory. At least I am assuming that.

Oh, I hadn't realized that, you're right. That is, for the file browser, at least. You can always provide files in different directories through the CLI. And of course the filetype case you mentioned also holds.